dev-docs/modules/userId.md

```diff
@@ -96,7 +96,7 @@ userIds: [
   }
 ]
 ```
-The Rubicon bid adapter would then receive 
+The Rubicon bid adapter would then receive
```

Comment:
same comment as above

Reply:
no actual changes were performed on the text e.g. nothing added or remoted in any way. I believe the IDE marked some changes when text was cut and pasted from between sections; I'll redo the file with the insertion of our submodule description only